worlds/sc2/Rules.py

```diff
@@ -929,6 +930,7 @@ def __init__(self, world: World):
         self.kerrigan_levels_per_mission_completed = get_option_value(world, "kerrigan_levels_per_mission_completed")
         self.kerrigan_levels_per_mission_completed_cap = get_option_value(world, "kerrigan_levels_per_mission_completed_cap")
         self.kerrigan_total_level_cap = get_option_value(world, "kerrigan_total_level_cap")
+        self.morphling_enabled = get_option_value(world, "enable_morphling")
```

Comment:
I'd add `== EnableMorphling.option_true` so `self.morphling_enabled` would be a proper boolean

Reply:
Good catch, thank you!